packages/tools/u-boot/package.mk

```diff
@@ -28,8 +28,8 @@ case "$PROJECT" in
     PKG_PATCH_DIRS="rockchip"
     ;;
   *)
-    PKG_VERSION="2019.04"
-    PKG_SHA256="76b7772d156b3ddd7644c8a1736081e55b78828537ff714065d21dbade229bef"
+    PKG_VERSION="2019.07"
+    PKG_SHA256="bff4fa77e8da17521c030ca4c5b947a056c1b1be4d3e6ee8637020b8d50251d0"
```

Comment:
This change can be dropped now, it's in master already: https://github.com/LibreELEC/LibreELEC.tv/pull/3756